docs/site/homepage/data/en/supported_apps.yml

```diff
@@ -0,0 +1,190 @@
+
+    # portfolio item loop
+    - name : Krita
+      image : images/supported_apps/krita.png
+      categories : ["2D", "Paint"]
+      content : "Version 2.6 and up."
+      link : "http://www.krita.org/"
+
+    # portfolio item loop
+    - name : Mari
+      image : images/supported_apps/mari.png
+      categories : ["3D", "Paint", "Texture"]
+      content : Version 1.4 and up.
+      link : "http://www.thefoundry.co.uk/products/mari"
+
+    # portfolio item loop
+    - name : Maya
+      image : images/supported_apps/maya.png
+      categories : ["3D", "Animation", "Modeling", "Lighting", "Texture"]
+      content : Maya 2016 and up.
+      link : "https://www.autodesk.com/products/maya/overview"
+
+    # portfolio item loop
+    - name : Mocha Pro
+      image : images/supported_apps/mocha.png
+      categories : ["3D", "Tracking"]
+      content : Mocha 2020 and up.
+      link : "https://borisfx.com/products/mocha-pro"
+
+    # portfolio item loop
+    - name : Modo
+      image : images/supported_apps/modo.png
+      categories : ["3D", "Modeling", "Texture", "Render"]
+      content : Maya 2016 and up.
+      link : "https://www.autodesk.com/products/maya/overview"
+
+    # portfolio item loop
+    - name : mrViewer
+      image : images/supported_apps/mrviewer.png
+      categories : ["Playback", "Review"]
+      content : Native support.
+      link : "https://mrviewer.sourceforge.io/"
+      
+    # portfolio item loop
+    - name : Nuke
+      image : images/supported_apps/nuke.png
+      categories : ["2D", "Compositing"]
+      content : Native support.
+      link : "http://www.thefoundry.co.uk/products/nuke"
+
+    # portfolio item loop
+    - name : OpenImageIO
+      image : images/supported_apps/oiio.png
+      categories : ["2D", "Photography", "Texture"]
+      content : Available in current code.
+      link : "http://openimageio.org/"
+
+    # portfolio item loop
+    - name : Adobe Photoshop
+      image : images/supported_apps/ps.png
+      categories : ["2D", "Photography", "Paint"]
+      content : Available via ICC profiles.
+      link : "https://www.adobe.com/products/photoshop.html"
+
+      # portfolio item loop
+    - name : PhotoFlow
+      image : images/supported_apps/photoflow.png
+      categories : ["2D", "Photography", "Paint"]
+      content : Native support.
+      link : "https://github.com/aferrero2707/PhotoFlow"
+
+    # portfolio item loop
+    - name : RV
+      image : images/supported_apps/rv.png
+      categories : ["Review"]
+      content : Native support.
+      link : "http://www.tweaksoftware.com/"
+
+    # portfolio item loop
+    - name : Silhouette
+      image : images/supported_apps/silhouette.png
+      categories : ["2D", "Roto", "Paint"]
+      content : Version 4.5 and up.
+      link : "http://www.silhouettefx.com/silhouette"
+
+    # portfolio item loop
+    - name : Adobe Substance
+      image : images/supported_apps/substance.png
```

Comment:
One small note, which can be addressed later. This Substance logo is the Substance Painter logo. Currently only Substance Designer has OCIO support. We could either swap in the Substance designer logo, or the more neutral Substance logo (no shape around the "S").